engines/griffon/cutscenes.cpp

```diff
@@ -133,6 +139,27 @@ const char *story2[27] = {
 	"and I am free to die as I please."
 };
 
+#ifdef USE_TTS
+int textToSpeech(int nextparagraph, const char *story[], int arraysize) { 
+Common::TextToSpeechManager *_ttsMan = g_system->getTextToSpeechManager();
+if (_ttsMan != nullptr && ConfMan.getBool("tts_enabled")) {
+	if ((_ttsMan->isSpeaking()) == false) {
+		Common::String paragraph;
+		while (nextparagraph < arraysize && story[nextparagraph][0] != ' ') {
+			paragraph += story[nextparagraph];
+			paragraph += " ";
```

Comment:
Doing it this way will leave a trailing space at the end of the prarapgaph. This probably doesn't matter, but this can be easily fixed by moving this at the start of the while loop if the parapraph is not empty:
```c++
	while (nextparagraph < arraysize && story[nextparagraph][0] != ' ') {
		if (!paragraph.empty())
			paragraph += " ";
		paragraph += story[nextparagraph++];
	}
```